gen/indykite/identity/v1beta1/attributes.ext.go

```diff
@@ -227,7 +226,3 @@ func (x PropertyBatchOperations) PreValidate(isTrusted bool) ([]string, []string
 	}
 	return replaced, removed, nil
 }
-
```

Comment:
Why you removed this? If it is missing `Error` function, it cannot be used in `error` interface.

Reply:
When I opened the sdk, this gave an error. Then I check in `jarvis-proto` and saw that @marvinkite removed this there in a recent PR.
You think it's wrong?